### DISCUSSION TITLE
coinbase wallet issue

### DISCUSSION THREAD

**Discussion by partikshadubb** [WalletConnect/walletconnect-monorepo#3964] [Developer Support] [answered]
i m facing an issue in @walletconnect/modal-react-native package , coinbase wallet i missing in wallet modal ,i want to know why coinbase walllet is missing in modal,please help us with this asap

**Comment by glitch-txs** [comment, marked as answer]
they don't support WalletConnect protocol, they have their own SDK. We're planning to add support for Coinbase in React Native Web3Modal SDK
